README.md

```diff
@@ -85,62 +87,70 @@ The following Invoke and Query APIs are available from both CLI and REST, and ha
 
 ### Invoke
-                * PostUser 
-                * PostItem
-                * PostAuctionRequest
-                * PostTransaction
-                * PostBid
-                * OpenAuctionForBids
-                * BuyItNow
-                * TransferItem
-                * CloseAuction
-                * CloseOpenAuctions
+                * iPostUser 
+                * iPostItem
+                * iPostAuctionRequest
+                * iPostTransaction
+                * iPostBid
+                * iOpenAuctionForBids
+                * iBuyItNow
+                * iTransferItem
+                * iCloseAuction
+                * iCloseOpenAuctions
 ### Query
-                * GetItem
-                * GetUser
-                * GetAuctionRequest
-                * GetTransaction
-                * GetBid
-                * GetLastBid
-                * GetHighestBid
-                * GetNoOfBidsReceived
-                * GetListOfBids
-                * GetItemLog
-                * GetItemListByCat
-                * GetUserListByCat
-                * GetListOfItemsOnAuc
-                * GetListOfOpenAucs
-                * ValidateItemOwnership
-                * IsItemOnAuction
+                * qGetItem
+                * qGetUser
+                * qGetAuctionRequest
+                * qGetTransaction
+                * qGetBid
+                * qGetLastBid
+                * qGetHighestBid
+                * qGetNoOfBidsReceived
+                * qGetListOfBids
+                * qGetItemLog
+                * qGetItemListByCat
+                * qGetUserListByCat
+                * qGetListOfItemsOnAuc
+                * qGetListOfOpenAucs
+                * qValidateItemOwnership
+                * qIsItemOnAuction
 
 ##Environment Setup
 
 Please review instructions on setting up the [Development Environment](https://github.com/hyperledger/fabric/blob/master/docs/dev-setup/devenv.md) as well as the setting up the [Sandbox Environment](https://github.com/hyperledger/fabric/blob/master/docs/Setup/Chaincode-setup.md) to execute the chaincode.
 
 ## Running the Application
 
-**Cloning the auction app**
+** Fork and clone the fabric**
 ```
+ git clone https://github.com/hyperledger/fabric.git
  cd $GOPATH/src/github.com/hyperledger/fabric
+ Fork Auction git repository from here https://github.com/ITPeople-Blockchain/auction.git and clone from your remote auction repository
  git clone https://github.com/<username>/auction.git
 ```
 
 ###Terminal 1
+
+Start Peer
+
 ```
  cd $GOPATH/src/github.com/hyperledger/fabric
- make peer
- peer node start --peer-chaincodedev
+ make peer && make orderer
+ peer node start -o 127.0.0.1:7050
 ```
 
 ###Terminal 2
+
+Start Orderer
 ```
- cd $GOPATH/src/github.com/hyperledger/fabric/auction/art/artchaincode
- go build
- CORE_CHAINCODE_ID_NAME=mycc CORE_PEER_ADDRESS=0.0.0.0:30303 ./artchaincode dev
+ cd $GOPATH/src/github.com/hyperledger/fabric
+ orderer

```

Comment:
**orderer** should be started first even before you start peer

also it is good to mention the below steps as first step
clear all production data with the command `rm -rf /var/hyperledger/*`